docs/documentation/cn/sql-reference/sql-functions/aggregate-functions/var_pop.md

```diff
@@ -0,0 +1,19 @@
+# VAR_POP
```

Comment:
VAR_POP and VARIANCE_POP are same, you can write them together, other VAR** as well